packages/server/lib/controllers/proxy.coffee

```diff
@@ -100,9 +91,6 @@ module.exports = {
   getHttpContent: (thr, req, res, remoteState, config, request) ->
     process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
 
-    ## prepends req.url with remoteState.origin
-    remoteUrl = req.proxiedUrl
```

Comment:
curious what your thought process was removing this

Reply:
`req.proxiedUrl` was used in some places and `remoteUrl` in others, figured it was a potential point of confusion ("wait, is proxiedUrl the same as remoteUrl?") so I just normalized all the references to it